README.md

```diff
@@ -217,6 +217,30 @@ def context(
 
 When using that all pages inside your test are created from the persistent context.
 
+### Using with `unittest.TestCase`
```

Comment:
We could add this into the upstream docs. I think our readme should doc-free...

Reply:
Will probably follow up there.